storage/deliveryjobrepo.go

```diff
@@ -29,15 +29,15 @@ func (djRepo *DeliveryJobDBRepository) DispatchMessage(message *data.Message, de
 		err = ErrInvalidStateToSave
 	}
 	args := make([]interface{}, 0, len(deliveryJobs)*jobPropertyCount)
-	query := "INSERT INTO job (id, messageId, consumerId, dispatchReceivedAt, statusChangedAt, earliestNextAttemptAt, status, createdAt, updatedAt) VALUES"
+	query := "INSERT INTO job (id, messageId, consumerId, dispatchReceivedAt, statusChangedAt, earliestNextAttemptAt, status, createdAt, updatedAt, priority) VALUES"
 	if err == nil {
 		for _, job := range deliveryJobs {
 			if job == nil || !job.IsInValidState() || job.Message.ID != message.ID {
 				err = ErrInvalidStateToSave
 				break
 			}
-			args = append(args, job.ID, job.Message.ID, job.Listener.ID, job.DispatchReceivedAt, job.StatusChangedAt, job.EarliestNextAttemptAt, job.Status, job.CreatedAt, job.UpdatedAt)
-			query = query + " (?, ?, ?, ?, ?, ?, ?, ?, ?),"
+			args = append(args, job.ID, job.Message.ID, job.Listener.ID, job.DispatchReceivedAt, job.StatusChangedAt, job.EarliestNextAttemptAt, job.Status, job.CreatedAt, job.UpdatedAt, job.Message.Priority)
```

Comment:
Priority should be set when job is initialized so SQL insertion should happen from the job object itself. It is used in [NewDeliveryJob](https://github.com/newscred/webhook-broker/blob/306b8cb778eb882d7f9facc29d7378d96d72ef7f/storage/data/job.go#L114)

Reply:
Updated.